ckanext/unaids/theme/templates/package/activity.html

```diff
@@ -0,0 +1,76 @@
+{% extends "package/read_base.html" %}
+
+{% block subtitle %}{{ _('Activity Stream') }} {{ g.template_title_delimiter }} {{ super() }}{% endblock %}
+
+{% block primary_content_inner %}
+<h1 class="hide-heading">
+  {% block page_heading %}
+  {{ _('Activity Stream') }}
+  {% endblock %}
+</h1>
+{% snippet 'snippets/activity_stream.html', activity_stream=activity_stream, id=id, object_type='package' %}
+
+<link rel="stylesheet" href="/bootstrap-table.css" />
+<div id="ActivityStreamLoading" class="text-center text-muted">
+  <h2>
+    <i class="fa fa-spinner fa-spin"></i>
+    <span> {{ _('Loading') }}</span>
+  </h2>
+</div>
+<div id="ActivityStreamContainer">
+  <table class="activity">
+    <thead>
+      <tr>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody></tbody>
+  </table>
+</div>
+
+<style>
+  ul.activity {
+    display: none;
+  }
+
+  #ActivityStreamContainer td {
+    border: none;
+    padding-bottom: 15px;
+  }
+</style>
+
+<script>
```

Comment:
The js code should be a separate js file configured in webassets and later requested with:
```
{% asset 'xxx' %}
```